README.md

```diff
@@ -30,16 +30,21 @@ The pipeline exposes the following standard settings:
 In addition to this, the following custom settings are required for this
 pipeline, and come with default values:
 
-* `pipe_vessels.source_table`: BigQuery table to read scored messages from, used as the source for track aggregation. Defaults to `messages_scored_`
-* `pipe_vessels.bigquery_segment_vessel`: BigQuery table to read segment to vessel mappings. Defaults to `segment_vessel`
-* `pipe_vessels.bigquery_tracks`: BigQuery table to write aggregated tracks into. Defaults to `tracks`
+* `pipe_vessels.source_table`: BigQuery table to read scored messages from, used as the source for track aggregation. Defaults to `messages_scored_`.
+* `pipe_vessels.bigquery_segment_vessel`: BigQuery table to read segment to vessel mappings. Defaults to `segment_vessel`.
+* `pipe_vessels.bigquery_tracks`: BigQuery table to write aggregated tracks into. Defaults to `tracks`.
```

Comment:
@andres-arana mention here which is input and which is output table.

Reply:
@enriquetuya It's already saying that, "BigQuery table to **read** messages from", "BigQuery table to **write** aggregated tracks into, etc.